script/jenkins/build.sh

```diff
@@ -1,15 +1,5 @@
 #!/usr/bin/env bash
-set -euo pipefail
+set -xeuo pipefail
 
-source ./_beats/dev-tools/common.bash
-
-jenkins_setup
-
-cleanup() {
-  rm -rf $TEMP_PYTHON_ENV
-  make stop-environment fix-permissions
-}
-trap cleanup EXIT
-
-make clean
-make
+mage -debug clean
```

Comment:
use mage commands directly

Reply:
this isn't identical, `make clean` cleans up a bunch of other things including `build/` and `fields.yml`

Reply:
the workspace is clean, I can remove the clean too